.github/workflows/issue-link.yml

```diff
@@ -13,3 +13,31 @@ jobs:
           branch-prefix: 'issue-' # required
           resolve: 'true' # optional (default: "false")
           link-style: 'comment' # optional (default: "body")
+
```

Comment:
This doesn't work if added to the circleci.yaml file?

Reply:
As far as i understand it does not interfere with circleci.. if you know something else i should check let me know.